engine-standalone-nep141-legacy/src/fungible_token.rs

```diff
@@ -88,14 +94,14 @@ pub struct FungibleTokenMetadata {
-            reference: None,
-            reference_hash: None,
-            decimals: 18,
-        }
+			spec: "ft-1.0.0".to_string(),
```

Comment:
Something wrong with the formatting. Looks like a tab symbol?

Reply:
These changes have been made by you 😄